platform/kube/inject/inject.go

```diff
@@ -328,7 +319,8 @@ func injectIntoSpec(p *Params, spec *v1.PodSpec) {
 		Command: []string{"/bin/sh"},
 		Args: []string{
 			"-c",
-			fmt.Sprintf("sysctl -w kernel.core_pattern=%s/core.%%e.%%p.%%t && ulimit -c unlimited", EnvoyConfigPath),
+			fmt.Sprintf("sysctl -w kernel.core_pattern=%s/core.%%e.%%p.%%t && ulimit -c unlimited",
+				p.Mesh.DefaultConfig.ConfigPath),
```

Comment:
This is a minor bug fix: both core dumps and mount volume were not respecting `ConfigPath` field in the proxy config.